starport/services/networkbuilder/networkbuilder.go

```diff
@@ -80,6 +81,16 @@ type initOptions struct {
 	keyringBackend           chaincmd.KeyringBackend
 }
 
+// newInitOptions initializes initOptions
+func newInitOptions(chainID string) *initOptions {
+	o := &initOptions{}
```

Comment:
```suggestion
	o := initOptions{
	   homePath: filepath.Join(homePath, chainID),
	}
```

Reply:
We can also set the user given options here.

Which means func signature needs to be updated as `newInitOptions(chainID string, options []initOptions)`.

And we move the for loop inside to apply these options.

Also, this func does not need to return a pointer.